runtime/src/accounts_db.rs

```diff
@@ -2771,6 +3199,7 @@ impl AccountsDB {
         hashes: &[Hash],
         storage_finder: Option<StorageFinder<'a>>,
         write_version_producer: Option<Box<dyn Iterator<Item = u64>>>,
+        use_cache: bool,
     ) -> StoreAccountsTiming {
         let storage_finder: StorageFinder<'a> = storage_finder
             .unwrap_or_else(|| Box::new(move |slot, size| self.find_storage_candidate(slot, size)));
```

Comment:
now that, we can remove `find_storage_candiate` because it's unused, probably?

Reply:
Hmm I think it's still used in tests now, since `storage_finder` is only None along the `accounts_db.store() -> do_store() -> store_accounts_default() pipeline`, and `accounts_db.store()` is still used by tests